incubator/kafqa/Chart.yaml

```diff
@@ -1,5 +1,6 @@
 apiVersion: v1
```

Comment:
Wondering, since we are making a breaking change, should we also bump this to version 2 so that it's only Helm 3 compatible.

Reply:
Not a bad idea 👍